test/e2e/resources/conditionals.brs

```diff
@@ -53,3 +53,53 @@ end if
 aa = {}
 if foo > 5 then aa.value = ">5" else aa.value = 8
 print aa.value
+
+' if not with boolean
+if not true then
+    print "this will never print"
+end if
+
+if not false then
+    print "not false"
+end if
+
+' if not with an expression
+bar = "abc"
+if not bar = "def" then
+    print "bar does not equal 'def'"
+end if
+
+' if not with or
+if not bar = "def" or false then
+    print "if not with or variation 1"
+end if
+
+if not bar = "abc" or true then
+    print "if not with or variation 2"
+end if
+
+' if not with and
+if not bar = "def" and true then
+    print "if not with and"
+end if
+
+' if not with two expressions
+foo = 1
+if not bar = "def" and foo = 1 then
```

Comment:
nit/not sure it matters but in the rest of this file, relational operators are used (e.g., `if foo < 5` on line 46) instead of equality operators (...`and foo = 1`).  I only bring it up bc of the specific change in Parser.ts (`let right = relational()`).

Reply:
Great callouts! I will definitely add tests for other operators 👍